runners/console/index.ts

```diff
@@ -92,6 +107,19 @@ export class Console extends Runner {
         console.log("assertCobiGenJava");
```

Comment:
let's properly console log that there is no assertion at all. 
We most properly need to address it later